contracts/core/Anchor.md

```diff
@@ -0,0 +1,63 @@
+# Anchor.sol
+
+The `Anchor` contract serves as a crucial utility within the Allo ecosystem, facilitating the execution of calls to target addresses. Anchors are associated with profiles and are accessible exclusively to the profile owner. This contract ensures secure and authorized interaction with external addresses, enhancing the capabilities of profiles and enabling controlled execution of operations. The contract leverages the `Registry` contract for ownership verification and access control.
+
+## Table of Contents
+- [Anchor.sol](#anchorsol)
+  - [Table of Contents](#table-of-contents)
+  - [Smart Contract Overview](#smart-contract-overview)
+    - [Storage Variables](#storage-variables)
+    - [Errors](#errors)
+    - [Constructor](#constructor)
+    - [External Functions](#external-functions)
+    - [Actors](#actors)
+  - [User Flows](#user-flows)
+    - [Constructor](#constructor-1)
+    - [Execute Call to a Target Address](#execute-call-to-a-target-address)
+
+
+## Smart Contract Overview
+
+* **License:** The `Anchor` contract is licensed under the AGPL-3.0-only license, promoting the use of open-source software.
+* **Solidity Version:** Developed using Solidity version 0.8.19, harnessing the latest advancements in Ethereum smart contract technology.
+
+### Storage Variables
+
+1. `registry` (Public Immutable): A reference to the `Registry` contract instance, enabling access to profile ownership information and access control.
+2. `profileId` (Public Immutable): The profile ID associated with the anchor, used to verify the caller's ownership.
+
+### Errors
+
+1. `UNAUTHORIZED()`: An error triggered when an unauthorized caller attempts to execute a function reserved for the profile owner.
+2. `CALL_FAILED()`: An error triggered when a call to a target address fails or is 0.
+
+### Constructor
+
+The constructor initializes the `registry` variable with a reference to the `Registry` contract and assigns the provided profile ID to the `profileId` variable.
+
+### External Functions
+
+1. **`execute`**: Execute a call to a target address, sending a specified amount of native tokens and data. Only the profile owner can initiate this operation.
+
+### Actors
```

Comment:
do we need to add members here?

Reply:
no, since only the owner can execute functions on the anchor (like moving funds, etc..)
members do not have any rights